src/main/java/io/vertx/redis/impl/RedisClientImpl.java

```diff
@@ -34,7 +34,7 @@
 
 public final class RedisClientImpl extends AbstractRedisClient {
 
-  RedisTransaction transaction;
+  private RedisTransaction transaction;
```

Comment:
Can be considered as a breaking change. However, I think it's fine in this case.